docs/src/lib/TokenTable.jsx

```diff
@@ -17,11 +17,17 @@ export function TokenTable( { tokens } ) {
 				{
 					tokens.map( ( token ) => (
 						<tr key={ token.name } id={ token.name }>
-							<td className="token-name">
+							<td>
 								<AnchorMdx href={ '#' + token.name }>🔗</AnchorMdx>
-							&nbsp;<strong>{ token.name }</strong>
+							&nbsp;<strong className="token-name">{ token.name }</strong>
 								<div className="referenced-tokens" title="value influenced by">
-									{ token.referencedTokens || '—' }
+									{token.referencedTokens.length > 0 ?
+										token.referencedTokens.map( ( token, i ) => [
```

Comment:
Explaining this to myself (correct me if something is not right)...

This
* always yields two entries
* first entry is a text node adding the comma for all but the first (0th) item
* second entry has no surprises
* react, upon seeing this, puts both nodes into the DOM, kind of like a fragment[0]
  * I assume, because the first node is text-only some of the rules don't apply (no key, no shared parent between the two of them)

[0]: https://reactjs.org/docs/fragments.html